specification/cognitiveservices/data-plane/Face/stable/v1.0/Face.json

```diff
@@ -4175,13 +4184,21 @@
     },
     "RecognitionModel": {
       "name": "RecognitionModel",
-      "description": "Recognition model name, maximum length is 128.",
+      "description": "Recognition model name.\nRecognition model is used when the face features are extracted, so a recognition model version could be provided when performing a Detection.\n",
```

Comment:
1. Curious what is the trailing `\n` for.
1. Should we use `<br/>` here?

Reply:
Removed.